src/main/java/uk/gov/hmcts/reform/cwrdapi/util/JavaRecordFeat.java

```diff
@@ -0,0 +1,4 @@
+package uk.gov.hmcts.reform.cwrdapi.util;
```

Comment:
I have written this class to verify if Java 17 feature works as a POC. This change is not required as part of this ticket